blockly/build.gradle

```diff
@@ -0,0 +1,49 @@
+import org.gradle.nativeplatform.platform.internal.DefaultNativePlatform
+
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This is a general purpose Gradle build.
+ * To learn more about Gradle by exploring our Samples at https://docs.gradle.org/8.5/samples
+ */
+
+plugins {
+    id 'java'
+}
+
+java {
```

Comment:
kann das nicht in der konfig einmal für alle projekte erledigt werden? oder muss man das wirklich in jedem sub-gradle neu definieren?